packages/cli/src/config/AmplitudeConfigYamlParser.ts

```diff
@@ -52,3 +56,36 @@ export function isValid(config: AmplitudeConfigModel): ConfigValidation {
     errors: errors.length > 0 ? errors : undefined,
   };
 }
+
+export function loadLocalConfiguration(configPath: string): AmplitudeConfig {
+  console.log(`Config Path:`, configPath);
+
+  const ymlConfig = fs.readFileSync(path.resolve(configPath), 'utf8');
+
+  const configModel = parseFromYaml(ymlConfig);
+
+  const configValidation = isValid(configModel);
+  if (!configValidation.valid) {
+    throw new Error(`Error loading configuration from ${configPath}. ${configValidation.errors}`);
+  }
+
+  return new AmplitudeConfig(configModel);
+}
+
+export function saveYamlToFile(filePath: string, config: AmplitudeConfig) {
+  // Clean up variants prior to output
+  const outputConfig: AmplitudeConfigModel = cloneDeep(config.model)
+
+  const flagKeys = Object.keys(outputConfig.experiment.flags);
+  flagKeys.forEach(flagKey => {
+    const flag = outputConfig.experiment.flags[flagKey];
+    if (isEmpty(flag.description)) {
+      delete flag.description;
```

Comment:
Minor, not sure why we delete the key, is it possible to have empty string as `description`? (Not an expert of experiment flag)